### PR TITLE
Mention of QGIS fulltext search plugin

### LINKED ISSUE #72: Suggestion : Mention existence of qgis_geomapfish_locator
https://github.com/opengisch/qgis_geomapfish_locator

Useful for QGIS users.

### PR DESCRIPTION
**Pull request by ponceta**
Useful for many. To be translated once validated.

Should fix https://github.com/geomapfish/geomapfish_website/issues/72

### CODE REVIEW

File: content/pages/2_functionalities.md [hunk 1]
```diff
@@ -126,8 +126,17 @@ Read the documentation:
 * MapServer
 * QGIS Server
 * ArcGIS for Server
+* Fulltext search service
```
Comment (rbovard) [inline content/pages/2_functionalities.md]: I won't put this here, it was more to mention which apps are supported as map server.

File: content/pages/2_functionalities.md [hunk 2]
```diff
@@ -126,8 +126,17 @@ Read the documentation:
...
 ### ![Functionalities description]({filename}/images/3_3_docker.png) Docker deployment
 
 <br /><br />
 For a better overview between features and versions, please refer to the [feature page](https://github.com/camptocamp/ngeo/blob/master/docs/features.md).
+
+---
+
+## Others
+
+### QGIS GeoMapFish Locator Plugin
```
Comment (rbovard) [inline content/pages/2_functionalities.md]:
```suggestion
### QGIS plugin
```

File: content/pages/2_functionalities.md [hunk 3]
```diff
@@ -126,8 +126,17 @@ Read the documentation:
...
+
+* Use of GeoMapFish fulltext search service in [QGIS](https://plugins.qgis.org/plugins/geomapfish_locator/).
```
Comment (rbovard) [inline content/pages/2_functionalities.md]:
```suggestion
* [GeoMapFish Locator](https://plugins.qgis.org/plugins/geomapfish_locator/): use GeoMapFish full text search service in QGIS.
```

**Review by rbovard — CHANGES_REQUESTED** [review]
Thanks a lot @ponceta!

I added some little remarks, if you're ok with it you can start translate it.

**Comment by rbovard** [comment]
@ponceta Up ;)

File: content/pages/2_functionalities.md (hunk 1 above)
Comment (ponceta) [reply content/pages/2_functionalities.md]:
```suggestion
```

**Comment by ponceta** [comment]
Applied changes.

**Comment by rbovard** [comment]
Thanks a lot, you can translate it.

**Comment by rbovard** [comment]
@ponceta Up again ;)

**Comment by ponceta** [comment]
@rbovard Translations done! ;)

**Review by rbovard — APPROVED** [review]
Thanks @ponceta

**Comment by rbovard** [comment]
@pfanguin Do you have the ability to fix the conflicts and merge? I'm only on the web interface and the conflicts are too complex to resolve them in there...

**Comment by ponceta** [comment]
@rbovard I recently tried visual studio code + github for desktop and it is quite efficient! (less cli and more content update) Especially for documentations and websites hosted on git.

**Comment by rbovard** [comment]
Replaced by #95

**Pull request closed without merging**